crates/serpen/tests/fixtures/comprehensive_ast_rewrite/models/user.py

```
@@ -0,0 +1,188 @@
+        logger._log_process(f"Processing user data: {data}")
+        processed = f"model_user_processed: {data}"
+    else:
+        processed = "model_user_invalid_data"
+
+    # Update global result
+    result["last_process"] = processed
+
+    return processed
+
+
+def validate(user_data: Dict[str, Any]) -> bool:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)